server/drafty/drafty_test.go

```diff
@@ -164,7 +182,7 @@ func TestPreview(t *testing.T) {
 		}
 		res, err := Preview(val, 15)
 		if err == nil {
-			t.Errorf("invalid input %d did not cause an error '%s'", testsToFail[i], res)
+			t.Errorf("invalid input %d did not cause an error '%s'", i, res)
```

Comment:
It is incorrect to put `testsToFail[i]` here.

Eg: Take the invalidInputs[5] and correct it such that line:185 would throw error. Because value of `i` at this stage is 5, and testsToFail length is only 4, so it would `panic: runtime error: index out of range [5] with length 4`